src/components/menu/menu-popover.tsx

```diff
@@ -55,7 +57,7 @@ export const MenuPopover: FC<{
   }
 
   return (
-    <ElMenuPopover style={popoverStyle} ref={popoverRef} {...getPopoverProps()} onClick={handleClick}>
+    <ElMenuPopover style={popoverStyle} ref={popoverRef} {...getPopoverProps()} onClick={handleClick} {...props}>
```

Comment:
**issue:** technically speaking, we should really be passing all additional props to `getPopoverProps` so it can correctly handle prop collisions, but I don't believe it supports that yet. For now, we probably just need to ensure `props` are spread on the component first, then our other props last, otherwise a `props.onClick` function will override our `onClick` prop and things will break.

Reply:
ah yes 🤦 , I will update this, thanks.